docs/proofs/interactive.rst

```diff
@@ -25,145 +123,147 @@ hypothesis, and insert *holes* for the rest of the definition:
 
 .. code-block:: idris
 
+    import Pruviloj
+    import Pruviloj.Induction
+
     plusReducesZ' : (n:Nat) -> n = plus n Z
     plusReducesZ' Z     = ?plusredZ_Z
     plusReducesZ' (S k) = let ih = plusReducesZ' k in
                           ?plusredZ_S
 
 On running , two global names are created, ``plusredZ_Z`` and
-``plusredZ_S``, with no definition. We can use the ``:m`` command at the
-prompt to find out which holes are still to be solved (or, more
-precisely, which functions exist but have no definitions), then the
-``:t`` command to see their types:
+``plusredZ_S``, with no definition.
 
 .. code-block:: idris
 
-    *theorems> :m
-    Global holes:
-            [plusredZ_S,plusredZ_Z]
+    *theorems> : idris plusReducesZ.idr -p pruviloj
 
-.. code-block:: idris
+    .  /  _/___/ /____(_)____
+       / // __  / ___/ / ___/     Version 1.2.0
+     _/ // /_/ / /  / (__  )      http://www.idris-lang.org/
+    /___/\__,_/_/  /_/____/       Type :? for help
 
-    *theorems> :t plusredZ_Z
-    plusredZ_Z : Z = plus Z Z
+    Idris is free software with ABSOLUTELY NO WARRANTY.
+    For details type :warranty.
+    Holes: Main.plusredZ_S, Main.plusredZ_Z
 
-    *theorems> :t plusredZ_S
-    plusredZ_S : (k : Nat) -> (k = plus k Z) -> S k = plus (S k) Z
+This tells us that we have two holes Main.plusredZ_S and Main.plusredZ_Z. We can solve
+these separately, ``plusredZ_Z`` is the simplest so we will do that first.
 
-The ``:p`` command enters interactive proof mode, which can be used to
-complete the missing definitions.
+The ``:elab plusredZ_Z`` command enters interactive proof mode, which can be used to
```

Comment:
Instead of calling it "interactive proof mode", I'd call it "interactive elaboration mode".